CHANGELOG.rst

```diff
@@ -9,6 +9,11 @@ While in version ``0``, minor and patch upgrades converge in the ``patch`` numbe
 Changelog
 =========
 
+v0.6.8 (2022-06-01)
```

Comment:
No. You should not add the version number. That's something the CI will take care of :wink:
See here -> https://github.com/julie-forman-kay-lab/IDPConformerGenerator/blob/77a29644a1a907f21ddd93eb5757ba8ae1bbe305/.bumpversion.cfg#L30-L39

You need to follow these instructions:

https://github.com/julie-forman-kay-lab/IDPConformerGenerator/blob/main/CONTRIBUTING.rst#update-changelog

Reply:
My bad! Thanks for clarifying and your patience :)